dev/release/create-tarball.sh

```diff
@@ -133,4 +132,4 @@ gpg --armor --output ${tarball}.asc --detach-sig ${tarball}
 echo "Uploading to apache dist/dev to ${url}"
 svn co --depth=empty https://dist.apache.org/repos/dist/dev/arrow ${SOURCE_TOP_DIR}/dev/dist
```

Comment:
Should this be updated too?

Reply:
Not yet. There will be a separate PR after the repo is renamed for updating the release aspects.